shared/actions/tracker.js

```diff
@@ -196,12 +192,14 @@ export function registerIdentifyUi (): TrackerActionCreator {
       })
     })
 
-    createServer(
-      engine,
-      'keybase.1.identifyUi.delegateIdentifyUI',
-      'keybase.1.identifyUi.finish',
-      () => serverCallMap(dispatch, getState)
-    )
+    engine.setIncomingHandler('keybase.1.identifyUi.delegateIdentifyUI', (param: any, response: ?Object) => {
```

Comment:
this is the 'new' version of the server. i think this is more straight forward. we get a call asking for a session, we make one. we can stop it whenever we want (added to serverCallMap)